pkg/messaging/nats/pubsub.go

```diff
@@ -15,10 +15,11 @@ import (
 	broker "github.com/nats-io/nats.go"
 )
 
-const chansPrefix = "channels"
-
-// SubjectAllChannels represents subject to subscribe for all the channels.
-const SubjectAllChannels = "channels.>"
+const (
+	chansPrefix = "channels"
+	// SubjectAllChannels represents subject to subscribe for all the channels.
+	SubjectAllChannels = "channels.>"
```

Comment:
I would construct this one as `chansPrefix + ".>"`

Reply:
That is what is implemented currently across all brokers. Should we change?